src/lib/libembind.js

```diff
@@ -32,7 +32,13 @@ var LibraryEmbind = {
   $PureVirtualError: class extends Error {},
   $GenericWireTypeSize: {{{ 2 * POINTER_SIZE }}},
 #if EMBIND_AOT
-  $InvokerFunctions: '<<< EMBIND_AOT_OUTPUT >>>',
+  $InvokerFunctions: '<<< EMBIND_AOT_INVOKERS >>>',
+#if MODULARIZE == 'instance'
+  // embindAssignExports is called in the generated code after embind is
+  // initialized. It will link the bindings to the ES module exports.
+  $embindAssignExports: '<<< EMBIND_AOT_ASSIGN_EXPORTS >>>',
+  $embindAssignExports__postset: () => { addAtPostCtor('embindAssignExports();'); },
```

Comment:
I wonder if there is any point in have this be part of the library file at all?  Couldn't the code in link.py just emit the whole `embindAssignExports()` function and call `addOnPostCtor(embindAssignExports)` right after?

Reply:
Yeah, this just provided a place to inject the code. I can just append it to the JS. It will end up after the postamble.js though. Is that fine?

Reply:
It it works it seems maybe less hacky?

Reply:
It it works it seems maybe less hacky?